src/services/videoconference/logic/notification.js

```diff
@@ -0,0 +1,24 @@
+const sendEmailNotification = (app, emailAdresses, bbbUrl) => {
```

Comment:
bbbUrl should be the link to the scope overview of events, tools for courses, not a direct bbb login url :) but however it would be possible to create all of the urls...